classes/publication/maps/Schema.inc.php

```diff
@@ -115,12 +115,22 @@ protected function mapByProperties(array $props, Publication $publication, bool
                     if ($this->anonymize) {
                         $output[$prop] = [];
                     } else {
-                        $output[$prop] = Repo::author()->getSchemaMap()->summarizeMany($publication->getData('authors'));
+                        $authors = Repo::author()->getMany(
+                            Repo::author()
+                                ->getCollector()
+                                ->filterByPublicationIds([$publication->getId()])
+                                ->orderBy(Repo::author()->getCollector()::ORDERBY_SEQUENCE)
+                        );
+
+                        $output[$prop] = Repo::author()->getSchemaMap()->summarizeMany($authors);
```

Comment:
We shouldn't need to go back to the database for this. If there's a problem with the authors as they are retrieved by the publication DAO, we should fix it there, so that `$publication->getData('authors')` is all that's needed.

Reply:
This could be deprecated due to code changes.